### PR TITLE
[CopyResourcesScript] Add --target-device to ibtool invocation for XIBs #5282

### PR DESCRIPTION
**Pull request by champo**
Fixes https://github.com/CocoaPods/CocoaPods/issues/5282

Tested on a project with several dependencies with XIBs and an example project for a library with XIBs

Depends on https://github.com/CocoaPods/cocoapods-integration-specs/pull/69

### CODE REVIEW

**Comment by segiddins** [comment]
@efirestone does this look good to you?

**Comment by efirestone** [comment]
Yeah, totally looks good. Must have been an oversight that we didn't add those flags for XIBs when we added them for storyboards.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 * Ensure `Set` is defined before using it.  
   [Samuel Giddins](https://github.com/segiddins)
   [#5287](https://github.com/CocoaPods/CocoaPods/issues/5287)
+* Add --target-device to ibtool invocation for XIBs
```
Comment (segiddins) [inline CHANGELOG.md]: spacing is off

**Comment by segiddins** [comment]
Merged the integration specs PR -- this is 👍 once the changelog spacing is addressed. Thanks, @champo !

**Comment by champo** [comment]
@segiddins On it! do you usually ammend commits with those kind of changes? I'm also going to bump the commit the specs point to the latest master

**Comment by segiddins** [comment]
@champo amending is preferred but not required :)

**Comment by champo** [comment]
@segiddins done! is travis always failing?

**Comment by segiddins** [comment]
@champo It was, but should've been fixed by https://github.com/CocoaPods/CocoaPods/commit/881e464c109f76baf484ff79e489db2156bcd461

**Comment by champo** [comment]
well, it seems travis doesn't like me. I can't reproduce the failure locally.

**Pull request merged**